### PR TITLE
chore(auth-server): fix display of domain on session connection view

### PR DESCRIPTION
**Pull request by itsacoyote**
# Description

Fix display of domain on the session connect page in Auth Server

Screenshot 2024-12-11 at 10 24 10 AM

Screenshot 2024-12-11 at 10 24 23 AM

### CODE REVIEW

File: packages/auth-server/components/session/Metadata.vue [hunk 1]
```diff
@@ -23,7 +23,8 @@
     </h1>
     <p
       v-if="domain"
-      class="text-center border border-neutral-800 bg-neutral-800/50 mt-3 mx-auto w-max px-4 py-1 rounded-3xl"
+      class="line-clamp-1 w-fit text-center border border-neutral-800 bg-neutral-800/50 mt-3 mx-auto px-4 py-1 rounded-3xl"
```
Comment (JackHamer09) [inline packages/auth-server/components/session/Metadata.vue:26]: Let's not truncate it but break to new line.
I think it's important for security reasons to see full url

**Comment by JackHamer09** [comment]
Also are pnpm lock file changes expected?

File: packages/auth-server/components/session/Metadata.vue (hunk 1 above)
Comment (itsacoyote) [reply packages/auth-server/components/session/Metadata.vue]: Updated images in description to reflect the changes

**Comment by itsacoyote** [comment]
> Also are pnpm lock file changes expected?

I ran a `pnpm install` at some point to resolve some local issues, which is why the pnpm.lock is there.

**Review by JackHamer09 — DISMISSED** [review]

**Pull request closed without merging**